pinot-broker/src/main/java/org/apache/pinot/broker/requesthandler/BaseBrokerRequestHandler.java

```diff
@@ -1706,11 +1707,14 @@ static void validateRequest(PinotQuery pinotQuery, int queryResponseLimit) {
         }
         List<Expression> orderByList = pinotQuery.getOrderByList();
         if (orderByList != null) {
+          List<Expression> selectOperands = selectList.get(0).getFunctionCall().getOperands();
+          List<Expression> orderByOperands = new LinkedList<>();
```

Comment:
No need to create this extra list, doing per-value check should be better without creating extra garbage

Reply:
done.